### PR TITLE
Update happy.js - comments

### PR DESCRIPTION
**Pull request by tallalnparis4ev**
Added comments to avoid confusion. I personally got a bit confused with this program's functionality. Added a few comments to help out other contributors.

### CODE REVIEW

File: src/happy.js [hunk 1]
```diff
@@ -1 +1,2 @@
+//This is a program that doesn't take any input and logs the string literal "happy" to console
```
Comment (tc80) [inline src/happy.js]:
```suggestion
// This is a program that doesn't take any input and logs the string literal "happy" to console.
```

**Review by tc80 — CHANGES_REQUESTED** [review]
other than this change lgtm! 😄

File: src/happy.js (hunk 1 above)
Comment (tallalnparis4ev) [reply src/happy.js]: I agree with the whitespace at the beginning, however I really do believe that it's better to state the input of the program as another dev may be confused.

Comment (tc80) [reply src/happy.js]: Agreed. I edited the suggestion

Comment (tallalnparis4ev) [reply src/happy.js]: Thanks. Will try and make these changes when I get the time. 
ETA: a few days.

**Comment by tallalnparis4ev** [comment]
@tc80  I have made your requested changes.

I'm pretty sure they don't impact other areas of the project, could u just make sure of that?

**Review by tc80 — APPROVED** [review]

**Comment by tc80** [comment]
lgtm, thank you for your contribution! 😄

**Pull request merged**